.github/git_rules/pr_approver.py

```
@@ -1,11 +1,14 @@
 #!/usr/bin/env python3
 """
-PR 승인 파일
-PR 규칙을 통과한 경우 승인 메시지 작성
+리뷰어 승인 파일
+AI 코드 리뷰 결과를 바탕으로 PR 승인 여부 결정
 """
 
 import os
+import sys
```

Ruff (0.12.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)